packages/client/src/checkout/types/checkoutOrder.types.ts

```diff
@@ -78,4 +78,7 @@ export type CheckoutOrder = {
   shippingMode: ShippingMode;
   paymentIntentId?: PaymentIntent['id'];
   promotionEvaluationId?: PromotionEvaluationId;
+  customerId?: number;
```

Comment:
I don't think these types belong here. From what I understood they're used on the request to create the order and not the response.

Reply:
Changed to the post request. Let me know if this is what was intended.

Reply:
Now it looks ok!
But on further investigation I think I've missed something before. `customerId` should be part of both the `checkoutOrder` and `checkoutSession` (as you had previously but **only** for `customerId`)